jni/Application.mk

```diff
@@ -1,5 +1,5 @@
 #APP_ABI := all
-APP_ABI := armeabi armeabi-v7a x86 x86_64 arm64-v8a
+APP_ABI := armeabi-v7a x86 x86_64 arm64-v8a
 # For future consideration (needs testing):
 #APP_ABI += mips mips64
-
+APP_PLATFORM := android-23
```

Comment:
Are we gaining anything from this bump ? Also, reading the docs https://developer.android.com/ndk/guides/application_mk#app_platform it states that this should match the minSdkVersion , 23 is Android 6 but we support Android 5.1 , right ? If there's no gain from this being set as we, I don't see a reason for this to be break change by bumping this to 23

Reply:
No, we only support Android 6+ in MABS 7. Both 5 and 5.1 were dropped.

Reply:
So we're just introducing a breaking change without actually needing it on this package, right? What if tomorrow we need to fix something here, we'll have to deal with release branches and whatnot for no reason

Reply:
I understand your point, but to be honest, I don't see it as a problem. Here are my thoughts regarding this topic:
- We already have breaking changes in the `cordova-sqlite-storage` plugin for MABS 7.
- The current version used by MABS 6 has been a stable version.
- If we somehow have to deal with some fixes in the lib versions that support `cordova-sqlite-storage` for MABS 6 we can just create a brunch for that.
- Also, the changes introduced in this PR aren't a simple fix that we can simply port to MABS 6. IMHO, they should only be applied to the version that will be released with MABS 7, due to the eventual impacts that they may cause.